.readthedocs.yml

```
@@ -1,22 +1,24 @@
 version: 2
 
+# conda:
+#   environment: environment.yml
+
 build:
   os: ubuntu-22.04
   tools:
     python: "miniconda-latest"
   apt_packages:
+    - nodejs
+    - npm
     - libglx-mesa0
     - libgl1
     - xvfb
-  jobs:
-    pre_build:
-      # Generate the Sphinx configuration for this Jupyter Book so it builds.
-      - "jupyter-book config sphinx book/"
-
-conda:
-  environment: environment.yml
-
-sphinx:
-  builder: html
-  fail_on_warning: true
-  configuration: book/_config.yml
\ No newline at end of file
+  commands:
+    - conda env create --name festim-workshop-jupyter2 --file environment.yml
+    # - cd book
+    # - conda run -n festim-workshop-jupyter2 python -m jupyter book start
+    - conda run -n festim-workshop-jupyter2 bash -c "cd book && python -m jupyter book build --html"
+    - ls
+    - mkdir -p $READTHEDOCS_OUTPUT/html
+    - ls
+    - cp -r _build/html/* $READTHEDOCS_OUTPUT/html/
```

```suggestion
    - cp -r book/_build/html/* $READTHEDOCS_OUTPUT/html/
```

Adding this back in since `cd book` is now commented out :)